libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,275 @@
+.nav-marquee{
+  flex: 1 1 auto;
+  background-color: black;
+}
+
+.nav-marquee.light {
+  color: var(--text-color);
+  background-color: #FFF;
+}
+
+.nav-marquee .content-table{
+  border: 1px solid #E8E8E8;
+  background-color: #FAFAFA;
+}
+
+.nav-marquee .content-table .toc-container {
+  flex: 1 1 auto; 
+  padding: 24px;
+}
+
+.nav-marquee .content-table .toc-title {
+  font-size: var(--type-heading-xs-size);
+  margin: 0;
+  font-weight: bold;
+}
+
+.nav-marquee .toc-list {
+  display: flex;
+  flex-direction: column; 
+  gap: 24px;
+  margin-top: 24px;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.nav-marquee .toc-item {
+  min-width: 200px;
+  display: flex;
+  position: relative; 
+  justify-content: space-between;
+  background-color: #FFF;
+  border: 1px solid #4B4B4B;
+  border-radius: 4px;
+  padding: 24px;
+  cursor: pointer;
+  line-height: 1.5;
+}
+
+.nav-marquee .toc-item:hover {
+  box-shadow: 2px 4px 5px #0000001A;
+  border: 1px solid #1473E6;
+}
+
+.nav-marquee .toc-item:focus-within {
```

Comment:
I can't see the styles being applied on desktop. The desktop styles override these declarations.
https://github.com/adobecom/milo/pull/786/files#diff-3052973bc600cc270df3e74fc90fefef74cfa0de6424aea73b1b415ea91e58a7R220-R221

Also, I don't see a reason for using focus-within, hover is good enough for this use-case.